packages/terra-date-picker/src/react-datepicker/index.jsx

```diff
@@ -389,7 +389,15 @@ class DatePicker extends React.Component {
     document.addEventListener('keydown', this.handleKeydown);
   }
 
-  componentDidUpdate() {
+  componentDidUpdate(prevProps) {
+    const currentMonth = prevProps.selected && getMonth(prevProps.selected)
+    const nextMonth = this.props.selected && getMonth(this.props.selected)
+    if (prevProps.inline && currentMonth !== nextMonth) {
+      this.setPreSelection(this.props.selected)
+    }
+    if (prevProps.highlightDates !== this.props.highlightDates) {
+      this.setState({'highlightDates': getHightLightDaysMap(nextProps.highlightDates)})
```

Comment:
This should be  this.props.highlightDates

Reply:
updated here https://github.com/cerner/terra-framework/pull/877/commits/1b2b89d008050997c92f076c9d3738acc8036e2a.